case_mapping/uco/observable.py

```diff
@@ -1099,12 +1099,14 @@ def __init__(
 class FacetMobileDevice(FacetEntity):
     def __init__(
         self,
-        IMSI=None,
-        ICCID=None,
-        IMEI=None,
-        storage_capacity=None,
-        keypad_pin=None,
-        MSISDN=None,
+        IMSI: Optional[str] = None,
+        ICCID: Optional[str] = None,
+        ESN: Optional[str] = None,
+        IMEI: Optional[str] = None,
+        bluetooth_device: Optional[str] = None,
+        keypad_pin: Optional[str] = None,
```

Comment:
This is named `observable:keypadUnlockCode` in the ontology.  The name of this property should probably update.